### ISSUE TITLE
Allow configuration of SSH exec command

### ISSUE THREAD

**Issue by rbayliss** [chanzuckerberg/blessclient#107]
Hey there,
First of all, thanks for this incredible piece of software.  It works super well.  I'm using it to manage SSH credentials in an environment where we require MFA login for AWS, and using `aws-vault` to securely store credentials and handle input of the MFA code.  This works super well on the command line already, where you can run `aws vault exec home -- blessclient run`, and everything works great.  But now we'd like to distribute a configuration that allows people to transparently tunnel through the bastion using their SSH config.  To do this, we've been using `blessclient import-config` to generate the SSH config, then ask people to make a manual modification:

```
# Original Command:
# Match OriginalHost  bastion.host exec "blessclient run"
# Updated command:
Match OriginalHost  bastion.host exec "aws-vault exec home -- blessclient run"
```

In other words, we need them to manually modify the exec command to invoke aws-vault first to obtain the credentials before invoking blessclient.  It sure would be nice if we could distribute this change in an automated fashion.  What do you think about adding configuration for the exec command?  Something like this:

```yaml
ssh_config:
  bastions:
    - pattern: "bastion.host"
      user: username
      # Invoke aws-vault first to obtain credentials
      exec: aws-vault exec home -- blessclient run
      hosts:
        - pattern: 10.*.*.*
```

**Comment by edulop91** [comment]
@rbayliss sounds reasonable to me! How does #108 look?

**Comment by rbayliss** [comment]
Incredible.  Exactly what we needed.  Thanks!
